scripts/prepare-og-images.mjs

```diff
@@ -0,0 +1,23 @@
+import puppeteer from 'puppeteer';
+
+import * as dotenv from 'dotenv';
+
+import { libs } from '../src/libs.mjs';
+
+dotenv.config();
+
+try {
+    const browser = await puppeteer.launch();
+    const page = await browser.newPage();
+
+    for (let i = 0; i < libs.length; i++) {
+        const { id } = libs[i];
+        await page.goto(`http://localhost:3000/preview/${id}`, { waitUntil: 'networkidle0' });
+        await page.setViewport({ width: 1200, height: 630 });
+        await page.screenshot({ path: `public/og-images/${id}.jpg` })
```

Comment:
I understand that, but I don't like it. :)

Reply:
94fcdba80ad7e467d8ecc2e4a50e131d84e0fa25 moved to function :D

Reply:
nope, I'm talking about using headless browser :)